e2e-tests/tests/e2e_flows.py

```diff
@@ -0,0 +1,310 @@
+            "original_score": 0.3, "score": 0.6499999999999999, "textual_explanation": null, 
+            "score_context_improvement": 0.3499999999999999, "supportive_context_word": "driver", 
+            "validation_result": null 
+            }
+        }
+    ]
+    """
+    assert response_status == 200
+    assert equal_json_strings(expected_response, response_content)
+
+    analyzer_data = json.loads(response_content)
+
+    anonymizer_request = {
+        "text": analyzer_request["text"],
+        "transformations": {
+            "DEFAULT": {"type": "replace", "new_value": "ANONYMIZED"},
+            "US_DRIVER_LICENSE": {"type": "mask", "masking_char": "*", "chars_to_mask": 4, "from_end": True},
+            "PERSON": {"type": "replace", "new_value": "<PERSON>"}
+        },
+        "analyzer_results": analyzer_data
+    }
+
+    response_status, response_content = anonymize(json.dumps(anonymizer_request))
+
+    expected_response = (
+        """{"result":"<PERSON> drivers license is AC43****"}"""
+    )
+    assert response_status == 200
+    assert equal_json_strings(expected_response, response_content)
+
+
+@pytest.mark.api
+def test_given_a_correct_analyze_input_high_threashold_then_anonymize_partially():
+    analyzer_request = json.loads("""
+    {
+        "text": "John Smith drivers license is AC432223",
+        "language": "en",
+        "remove_interpretability_response": 1,
+        "score_threshold": 0.7
+    }
+    """)
+
+    response_status, response_content = analyze(json.dumps(analyzer_request))
+
+    expected_response = """
+    [
+        {"entity_type": "PERSON", "start": 0, "end": 10, "score": 0.85, "analysis_explanation": null}
+    ]
+    """
+    assert response_status == 200
+    assert equal_json_strings(expected_response, response_content)
+
+    analyzer_data = json.loads(response_content)
+
+    anonymizer_request = {
+        "text": analyzer_request["text"],
+        "transformations": {
+            "DEFAULT": {"type": "replace", "new_value": "ANONYMIZED"},
+            "US_DRIVER_LICENSE": {"type": "mask", "masking_char": "*", "chars_to_mask": 4, "from_end": True},
+            "PERSON": {"type": "replace", "new_value": "<PERSON>"}
+        },
+        "analyzer_results": analyzer_data
+    }
+
+    response_status, response_content = anonymize(json.dumps(anonymizer_request))
+
+    expected_response = (
+        """{"result":"<PERSON> drivers license is AC432223"}"""
+    )
+    assert response_status == 200
+    assert equal_json_strings(expected_response, response_content)
+
+
+@pytest.mark.api
+def test_given_a_correct_analyze_input_high_threashold_and_upported_entities_then_anonymize_partially():
```

Comment:
maybe you meant: test_given_a_correct_analyze_input_with_high_threshold_and_unmatched_entities_then_anonymize_partially